app/api/auth/agent-connect/callback/route.ts

```diff
@@ -19,7 +19,7 @@ export const GET = withSession(async function callbackRoute(req) {
     const userInfo = await proConnectAuthenticate(req);
     const agent = await getAgent(userInfo);
 
-    // await verifyAgentHabilitation(agent);
+    await verifyAgentHabilitation(agent);
 
```

Comment:
I wonder, should we allow an agent to not have a siret ? If we choose not to, it could be part of the checks done in `verifyAgentHabilitation`

Reply:
At first, we were checking:
• If the provider is MCP, the user must have a SIRET.

However, we need a SIRET regardless of the provider.
So, I extended this check to apply to all providers.

Reply:
Is it what you meant ?